cypress/support/EUDescribe.js

```diff
@@ -0,0 +1,10 @@
+// Allows tests to skipped outside the EU
+// Used currently to stop EU specific tests
+// from running on servers in the US
+const EUDescribe = (name, func) => {
```

Comment:
If I'm being reeeeally pinickity I'd rename as `describeForEuOnly` as it's more readable - and I'd change the comment above to:

```
/**
 * Travis servers are based in USA so EU-specific behaviour cannot be tested.
 * Therefore skip these tests on Travis.
 */
```

Reply:
done :)